modules/post-install-edge-aws-extend-cluster.adoc

```diff
@@ -1,21 +1,67 @@
 // Module included in the following assemblies:
 //
-// * post_installation_configuration/aws-compute-edge-tasks.adoc
+// * post_installation_configuration/aws-compute-edge-tasks-zones
 
 :_mod-docs-content-type: CONCEPT
 [id="post-install-edge-aws-extend-cluster_{context}"]
-= Extend existing clusters to use AWS Local Zones
+= Extend existing clusters to use AWS zones
 
-As a postinstallation task, you can extend an existing {product-title} cluster on Amazon Web Services (AWS) to use AWS Local Zones.
+As a post-installation task, you can extend an existing {product-title} cluster on Amazon Web Services (AWS) to use AWS {zone-type}.
 
-Extending nodes to Local Zone locations comprises the following steps:
-
-- Adjusting the cluster-network maximum transmission unit (MTU)
-- Opting in the Local Zone group to AWS Local Zones
-- Creating a subnet in the existing VPC for a Local Zone location
-- Creating the machine set manifest, and then creating a node in each Local Zone location
+Extending nodes to {zone-type} locations comprises the following steps:
 
+- Adjusting the cluster-network maximum transmission unit (MTU).
+- Opting in the {zone-type} group to AWS {zone-type}.
+- Creating a subnet in the existing VPC for a {zone-type} location.
++
 [IMPORTANT]
 ====
-Before you extend an existing {product-title} cluster on AWS to use Local Zones, check that the existing VPC contains available Classless Inter-Domain Routing (CIDR) blocks. These blocks are needed for creating the subnets.
+Before you extend an existing {product-title} cluster on AWS to use {zone-type}, check that the existing VPC contains available Classless Inter-Domain Routing (CIDR) blocks. These blocks are needed for creating the subnets.
 ====
++
+- Creating the machine set manifest, and then creating a node in each Local Zone or Wavelength Zone location.
+- You added a permission for creating network resources that support AWS {zone-type} to the Identity and Access Management (IAM) user or role. For example:
++
+.Example of an additional IAM policy that attached `ec2:ModifyAvailabilityZoneGroup`, `ec2:CreateCarrierGateway`, and `ec2:DeleteCarrierGateway` permissions to a user or role for a Local Zone
```

Comment:
Incorrect statement for Local Zones. LZ does not support Carrier gateways so don't need to add permissions to manipulate that kind of resources. 

If you review the original source there are statements to isolate by zone-types, please isolate it according to the new format.

https://github.com/openshift/openshift-docs/pull/69128/files#diff-92599038675fb48d0094487829ff14c7e28123fb6302ffb15442e9dc9e3a1d86R10-R56

Reply:
I attempted to reduce creating a separate doc book for day-2 WZ tasks, as it might be excessive given that the majority of LZ and QZ info is shared. 

Would specifically stating WZ around the example help or do you think having two books would help? The [Post-installation](https://docs.openshift.com/container-platform/4.14/post_installation_configuration/index.html) section of the OCP docs needs an overhaul as it's a large mix of different platform tasks. Some discussion has been in place to move the platform-specific Post-installation documents to their [Installing section](https://docs.openshift.com/container-platform/4.14/installing/installing_aws/preparing-to-install-on-aws.html) of the customer docs.

Reply:
![Screenshot from 2024-01-23 10-52-50](https://github.com/openshift/openshift-docs/assets/57954076/5f892f45-5cc6-4b35-982e-60fe667bcb6d)